### PR TITLE
Nixos module

### PR DESCRIPTION
**Pull request by name-snrl**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -111,6 +111,7 @@
             inherit frontend;
           };
 
+          nixosModule = builtins.scopedImport { inherit gtchServer; } ./module.nix;
```
Comment (cab404) [inline flake.nix]: nixosModules.default, https://nixos.wiki/wiki/Flakes#Output_schema

Comment (name-snrl) [reply flake.nix]: Is it necessary? I did according to the template from NixOS repo:

https://github.com/NixOS/templates/blob/master/full/flake.nix#L136

Comment (cab404) [reply flake.nix]: templates are not being updated for a really long time now)

**Comment by cab404** [comment]
Took me a while :D

**Pull request merged**